ava-test/echo.js

```diff
@@ -129,3 +129,9 @@ test.cb('piping to a file with -n', t => {
     t.end();
   });
 });
+
+test('stderr with unrecognized options is null', t => {
+  const ret = shell.echo('-asdf');
```

Comment:
Yes it does. I suppose we can fix this along with mocking `process.stdout` like you suggested for #622.